.github/workflows/test.yaml

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: © 2024 Rajul Jha <rajuljha49@gmail.com>
```

Comment:
When using SPDX file tags, the © symbol is redundant. There is no hard rule on its use, but by not using you can keep your file clear of Unicode characters. Feel free to ignore this. Not critical.